source/channel.c

```diff
@@ -37,6 +37,7 @@ enum aws_channel_state {
     AWS_CHANNEL_ACTIVE,
     AWS_CHANNEL_SHUTTING_DOWN,
     AWS_CHANNEL_SHUT_DOWN,
+    AWS_CHANNEL_DESTROY_ONCE_REFCOUNT_0,
```

Comment:
[trivial] DESTROY_ON_REFCOUNT_0 reads better, but that's just my opinion, maaaaaannn